src/coreclr/interpreter/intops.def

```diff
@@ -403,6 +403,12 @@ OPDEF(INTOP_GC_COLLECT, "gc.collect", 1, 0, 0, InterpOpNoArgs)
 
 OPDEF(INTOP_LOAD_FRAMEVAR, "load.framevar", 2, 1, 0, InterpOpNoArgs)
 
+// Intrinsics
+OPDEF(INTOP_METHODTABLE, "methodtable", 3, 1, 1, InterpOpInt)
```

Comment:
can we name this INTOP_GETMETHODTABLE / getmethodtable for better clarity?

Reply:
Do we even need this opcode ? Shouldn't this be just a `LDIND_I` with offset 0 ?

Reply:
No need, replaced with LDIND_I. Thanks.